modules/apps/frontend-js/frontend-js-spa-web/npmscripts.config.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+module.exports = {
+	federation: {
+		bridges: ['senna'],
```

Comment:
I don't think this bridge is needed. If I understand correctly then, this would provide a Loader2Webpack bridge for anyone doing something like `import senna from 'senna'` and compiling using bundler-v2.

If that's correct and since we know we don't have such a scenario, I would avoid this.

Reply:
Yes, I deleted it but somehow I managed to revert the deletion at some point.